lib/routes/drivers/tests/routes.test.js

```diff
@@ -0,0 +1,167 @@
+    lastName: 'Wick',
+    email: 'wick@jhon.is',
+    mobileNo: '8585959524',
+    cabTypeId: 1,
+    cabNumber: 'MH 20 9211',
+    gender: 'male'
+};
+
+const payloadForLocation = {
+    driverId: 1,
+    currentLocation: [18.52043, 73.856743],
+    status: 'active'
+};
+
+const payloadForAccepting = {
+    driverId: 1,
+    bookingAttemptId: 1
+};
+
+describe('/drivers route  tests', () => {
+    let server;
+    beforeEach(async () => {
+        server = await resetAndMockDB(async allDbs => {});
+    });
+
+    it('should return 200', async () => {
+        const res = await server.inject({
+            method: 'POST',
+            url: '/drivers',
+            payload
+        });
+        expect(res.statusCode).toEqual(200);
+        const { result } = res;
+        expect(result.first_name).toEqual(payload.firstName);
+        expect(result.last_name).toEqual(payload.lastName);
+        expect(result.email).toEqual(payload.email);
+        expect(result.mobile_no).toEqual(payload.mobileNo);
+        expect(result.gender).toEqual(payload.gender);
+    });
+
+    it('should return 400', async () => {
+        const res = await server.inject({
+            method: 'POST',
+            url: '/drivers'
+        });
+        expect(res.statusCode).toEqual(400);
+    });
+
+    it('should return 400', async () => {
+        server = await resetAndMockDB(async allDbs => {
+            allDbs.drivers.create = () =>
+                new Promise((resolve, reject) => {
+                    reject(new Error('Test Error'));
+                });
+        });
+
+        const res = await server.inject({
+            method: 'POST',
+            url: '/drivers',
+            payload
+        });
+        expect(res.statusCode).toEqual(400);
+        expect(res.result.message).toEqual('Test Error');
+    });
+
+    it('should return 200', async () => {
+        const res = await server.inject({
+            method: 'PUT',
+            url: '/drivers/profile/1',
+            payload
+        });
+        expect(res.statusCode).toEqual(200);
+    });
+
+    it('should return 400', async () => {
+        const res = await server.inject({
+            method: 'PUT',
+            url: '/drivers/profile/1',
+            payload: {}
+        });
+        expect(res.statusCode).toEqual(400);
+        expect(res.result.message).toEqual('Invalid request payload input');
+    });
+
+    it('should return 400', async () => {
+        server = await resetAndMockDB(async allDbs => {
+            allDbs.drivers.update = () =>
+                new Promise((resolve, reject) => {
+                    reject(new Error('Test Error'));
+                });
+        });
+
+        const res = await server.inject({
+            method: 'PUT',
+            url: '/drivers/profile/1',
+            payload
+        });
+        expect(res.statusCode).toEqual(400);
+        expect(res.result.message).toEqual('Test Error');
+    });
+
+    it('should return 200', async () => {
```

Comment:
it should return 200 when <<this>> payload is provided or something similar, a little detail of what this test is for

Reply:
Done.